onnx/defs/object_detection/defs.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "onnx/defs/schema.h"
+using namespace ONNX_NAMESPACE;
+
+namespace ONNX_NAMESPACE {
+
+static const char* RoIAlign_ver1_doc = R"DOC(
+Region of Interest (RoI) align operation described in the
+[Mask R-CNN paper](https://arxiv.org/abs/1703.06870).
+RoIAlign consumes an input tensor X and region of interests (rois)
+to apply pooling across each RoI; it produces a 4-D tensor of shape
+(num_rois, C, output_height, output_width).
+
+RoIAlign is proposed to avoid the misalignment by removing
+quantizations while converting from original image into feature
+map and from feature map into RoI feature; in each ROI bin,
+the value of the sampled locations are computed directly
+through bilinear interpolation.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    RoIAlign,
```

Comment:
minor: capitalization of I. exiting MaxRoiPool op does not capitalize the I. should keep it consistent